### ISSUE TITLE
Error loading dio 8 with Brunch

### ISSUE THREAD

**Issue by aMarCruz** [thysultan/dio.js#36]
@thysultan ,
requiring dio 8 with [brunch](http://brunch.io) generates the following error:

```
Error: Cannot find module 'dio.js/dist/node' from 'dio.js/dist/umd.js'
Something went wrong trying to import the server module.
```

I don't use SSR so I can ignore the error, but it is annoying for my users.
Dio v7.x does not have this issue.

Anyway, this is a great library. Thanks for your work!

**Comment by thysultan** [comment]
Thanks. I didn't consider brunch, might also need to look at how fusebox bundles as well, but from the looks of it i have a patch in sight.

**Comment by thysultan** [comment]
Patched in [8.0.3](https://github.com/thysultan/dio.js/releases/tag/8.0.3)

**Comment by aMarCruz** [comment]
oh that was fast! thanks.
